### PR TITLE
Jenkins hygieia

### PR DESCRIPTION
**Pull request by etsauer**
#### What is this PR About?
Adding a custom Jenkins Master image that builds and installs the Hygieia pipeline plugin

#### How do we test this?
See jenkins-masters/hygieia-plugin/README.adoc

cc: @redhat-cop/containerize-it

### CODE REVIEW

File: jenkins-masters/hygieia-plugin/.applier/group_vars/seed-hosts.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    params_from_vars:
+      NAMESPACE: "{{ namespace }}"
+- object: Builds
+  content:
+  - name: Jenkins Master Image
+    template: "{{ inventory_dir }}/../.openshift/templates/jenkins-s2i.yml"
+    namespace: "{{ namespace }}"
+    params_from_vars:
+      JENKINS_GIT_URL: https://github.com/etsauer/containers-quickstarts.git
+      JENKINS_GIT_BRANCH: jenkins-hygieia
```
Comment (pcarney8) [inline jenkins-masters/hygieia-plugin/.applier/group_vars/seed-hosts.yml]: dont forget to change this back to master

File: jenkins-masters/hygieia-plugin/.openshift/templates/jenkins-ephemeral.yml [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    - name: web
+      nodePort: 0
+      port: 80
+      protocol: TCP
+      targetPort: 8080
+    selector:
+      name: ${JENKINS_SERVICE_NAME}
+    sessionAffinity: None
+    type: ClusterIP
+- kind: ConfigMap
```
Comment (pcarney8) [inline jenkins-masters/hygieia-plugin/.openshift/templates/jenkins-ephemeral.yml]: this is in two places

File: jenkins-masters/hygieia-plugin/jenkins-s2i/configuration/org.jenkinsci.plugins.workflow.libs.GlobalLibraries.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (pcarney8) [inline jenkins-masters/hygieia-plugin/jenkins-s2i/configuration/org.jenkinsci.plugins.workflow.libs.GlobalLibraries.xml]: is this file needed?

**Comment by etsauer** [comment]
[Jenkins Master with Hygieia Plugin](https://trello.com/c/Im3yUtlO/1-jenkins-master-with-hygieia-plugin)
